js_modules/dagit/packages/core/src/nav/RepoNavItem.tsx

```diff
@@ -130,16 +130,16 @@ const SingleRepoSummary: React.FC<{repoAddress: RepoAddress}> = ({repoAddress})
                   </Reloading>
                 }
               >
-                {reloading ? (
-                  <Box padding={2}>
+                <SpinnerOrButton>
```

Comment:
This is a bit non-intuitive since this sets display: flex and there's a single element child (nothing to flex really?) - is it sufficient to have a plain div here giving a block layout?